modules/ROOT/pages/footnotes.adoc

```diff
@@ -0,0 +1,135 @@
+
+The footnotes section is automatically updated whenever a footnote is inserted, reordered, or removed within the {productname} editor. The footnotes section is automatically removed if all footnotes are removed.
+
+For example, adding a new footnote between already extant footnotes ^*1*^ and ^*2*^ results in the new footnote becoming footnote ^*2*^ and the previous footnote ^*2*^ automatically becoming footnote ^*3*^.
+
+Superscripted footnote numbers can also be dragged to a different position in the document. Footnote numbers and their relationship with the respective footnote entry are automatically updated when they are dragged to a different position.
+
+Each superscripted footnote number links to its respective footnote entry, and each footnote entry is prefixed with a caret — *^* — that links back to its respective superscripted footnote number. This allows for navigation between the editor content and the footnotes section.
+
+If the TinyMCE document is large enough to produce a scroll bar, clicking the caret prefixing a footnote entry scrolls the document back to the line containing said entry’s superscripted footnote number.
+
+NOTE: Although the viewport scrolls back to the line containing the footnote number, the insertion point is not changed. Users must move the insertion point by, for example, clicking the I-beam pointer somewhere in the now-scrolled document.
+
+
+=== CSS
+
+//// explanatory text here.
+
+[source,css]
+----
+include::example$css-codeblock/footnotes-plugin.css[]
+----
+
+
+== Usage notes
+
+=== The update footnotes button
+
+The update footnotes button, as its name implies, updates the state of all footnotes in a {productname} document.
+
+To add the update footnote button to the {productname} toolbar, add `footnotesupdate` to the `toolbar`.
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'footnotes',
+  toolbar: 'footnotes footnotesupdate',
+});
+----
+
+In the main, footnotes update automatically as they are added, removed, or edited. However, there are some circumstances where user-edits change footnote state but the footnotes themselves do not update.
+
+For example, when an image caption with an attached footnote is removed (by unchecking the _Show caption_ checkbox in the *Insert/Edit Image* dialog), the associated footnote text is not automatically removed.
+
+In similar fashion, if a table row or column that contains a footnote is removed, the associated footnote text is not automatically removed.
+
+These orphan footnote texts can be deleted manually. They can also be removed by clicking the `footnotesupdate` button in the toolbar.
+
+
+=== Footnotes and {productname} Previews
+
+*File > Preview* presents footnote-specific in-document links, but these links do not work in the Preview.
```

Comment:
I missed this the other day sorry, but should this be using the appropriate macros? It's something we've not had available for AsciiDoc. Maybe we should log a follow up task to improve this instead?

See https://docs.asciidoctor.org/asciidoc/latest/macros/ui-macros/

Reply:
Also likewise here, this isn't an issue with footnotes... it's an issue elsewhere and should not be documented here.

Reply:
> should this be using the appropriate macros?

A quick search through the docs suggests we’ve not used button and menu UI macros heretofore, so I’ve made this question a ticket: [DOC-1808](https://ephocks.atlassian.net/browse/DOC-1808).

Reply:
> Also likewise here, this isn't an issue with footnotes... it's an issue elsewhere and should not be documented here. If fact, links aren't supposed to work in preview anyways so this shouldn't even be a documented issue as it's generally expected for links to not work (though it could be argued in page links should work, but that's still a preview plugin bug not footnotes and applies to more than just footnotes).

Now a ticket: [DOC-1809](https://ephocks.atlassian.net/browse/DOC-1809).